src/app/todo/components/todoAdding/todo-adding.component..ts

```diff
@@ -62,6 +85,11 @@ export class TodoAddingComponent implements Partial<ITask> {
     });
   }
 
+  assignUser(user: string) {
+    this.assignedUsers.push(user);
+    debugger;
```

Comment:
debugger

Reply:
it'll be easier to debug through dev tools, no need to write debugger you can just use the source tab and set breakpoints there